cluster-sync/imageio/imageio.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: imageio-deployment
+  namespace: cdi
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: imageio
+  template:
+    metadata:
+      labels:
+        app: imageio
+    spec:
+      serviceAccountName: cdi-sa
+      containers:
+      - name: imageiotest
+        image: machacekondra/imageiotest:test
```

Comment:
What about Adam's comment to use kubevirt org for the tests containers?

Reply:
Well, that was one option. Second was to make it public, which I did:

 - https://github.com/machacekondra/imageiotest
 - https://github.com/machacekondra/fakeovirt